### PR TITLE
[SIW-1418] Add application insight configuration for Function App User v2

### PR DESCRIPTION
**Pull request by mamu0**
#### List of Changes

Create autoscaler.
Apply insights.
CODEOWNER minor change.

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: infra/resources/_modules/function_apps/function_app_user.tf [hunk 1]
```diff
@@ -24,6 +24,23 @@ module "function_app_user" {
 
   app_settings      = local.function_app_wallet.app_settings
   slot_app_settings = local.function_app_wallet.app_settings
+  sticky_app_setting_names = [
```
Comment (gunzip) [inline infra/resources/_modules/function_apps/function_app_user.tf:27]: i guess these settings shoud go into function module

Comment (Krusty93) [reply infra/resources/_modules/function_apps/function_app_user.tf:27]: I suppose we can get rid of all of them, as they look autogenerated by something.
@silvicir @grausof did you use any diagnostic feature from Azure Portal? Anything related to AI?

Some of them are documented [here](https://learn.microsoft.com/en-us/azure/azure-monitor/profiler/profiler), but they're out of our scope.

More info [here](https://stackoverflow.com/questions/77856491/application-insights-configuration-insanity)

Comment (grausof) [reply infra/resources/_modules/function_apps/function_app_user.tf:27]: We do not use any such function.

Comment (mamu0) [reply infra/resources/_modules/function_apps/function_app_user.tf:27]: So maybe i can remove them and check if they are generated automatically

Comment (Krusty93) [reply infra/resources/_modules/function_apps/function_app_user.tf:27]: They're not in configuration neither, they're only in TF state as sticky settings. We can remove them from code

**Review by grausof — APPROVED** [review]
LGTM

**Pull request merged**